torchvision/prototype/transforms/functional/_augment.py

```diff
@@ -17,15 +17,21 @@ def erase_image_pil(
     return to_pil_image(output, mode=image.mode)
 
 
+def erase_video(
+    video: torch.Tensor, i: int, j: int, h: int, w: int, v: torch.Tensor, inplace: bool = False
+) -> torch.Tensor:
+    return erase_image_tensor(video, i=i, j=j, h=h, w=w, v=v, inplace=inplace)
+
+
 def erase(
-    inpt: features.ImageTypeJIT,
+    inpt: features.InputTypeJIT,
```

Comment:
The proper type for of  `InputType` is defined as `Union[torch.Tensor, PIL.Image.Image, _Feature]`. Note that we include the base class for `_Feature` here not `Image`. So here we imply that BBoxes can be transformed (though I understand you use the JIT-friendly version but things go hand in hand).

One potential approach here is to add `Video` in the `ImageType`  and rename accordingly.

Reply:
What do you think would be a good name? `ImageOrVideoType`?

Reply:
I can live with that for now. We will rethink names before releasing.

Reply:
@pmeier You missed this comment. This should be `ImageOrVideoType` I think. Same on the return type.

Reply:
@pmeier You keep resolving this comment but it's still unresolved. This type is wrong, because BBoxes are not suppored. It needs to be `ImageOrVideoType`

Reply:
True, it seems I keep missing it :facepalm: